self_checkout_commandline.rb

```diff
@@ -0,0 +1,22 @@
+print "Enter the price of item 1:"
+item_one_price = gets.chomp.to_i
+print "Enter the quantity of item 1:"
+item_one_quantity = gets.chomp.to_i
+print "Enter the price of item 2:"
+item_two_price = gets.chomp.to_i
+print "Enter the quantity of item 2:"
+item_two_quantity = gets.chomp.to_i
+print "Enter the price of item 3:"
+item_three_price = gets.chomp.to_i
+print "Enter the quantity of item 3:"
+item_three_quantity = gets.chomp.to_i
```

Comment:
Could you use a loop to run this instead of running line by line?
The user should be able to put in as many items as they like, and then the loop will be broken when they choose to check out :)